test/dummy/app/models/content_type/validator/check.rb

```diff
@@ -27,6 +27,8 @@ def self.example_for(type, several: false)
   validates :extension_two_extensions_docx, content_type: 'application/vnd.openxmlformats-officedocument.wordprocessingml.document'
   has_one_attached :extension_two_extensions_pdf
   validates :extension_two_extensions_pdf, content_type: 'application/pdf'
+  has_one_attached :pdf_file
+  validates :pdf_file, content_type: 'application/pdf'
```

Comment:
To be consistent with the attribute naming, could you use `extension_upcase_extension` and `extension_missing_extension` as attribute names?
The change should also be reflected in `test/validators/content_type_validator_test.rb`

Reply:
@matthieuprat

Reply:
Done!

Reply:
Thanks a lot, all good to me!